### ISSUE TITLE
Documentation missing in ReadTheDocs

### ISSUE THREAD

**Issue by xavierhardy** [nickw444/flask-ldap3-login#7]
Hello Nick,

I get a 404 code back when I try to reach the ReadTheDocs page for the flask-ldap3-login.

http://flask-ldap3-login.readthedocs.org/en/latest/
```
You've found something that doesn't exist.

We're sorry, we don't know what you're looking for


        \          SORRY            /
         \                         /
          \    This page does     /
           ]   not exist yet.    [    ,'|
           ]                     [   /  |
           ]___               ___[ ,'   |
           ]  ]\             /[  [ |:   |
           ]  ] \           / [  [ |:   |
           ]  ]  ]         [  [  [ |:   |
           ]  ]  ]__     __[  [  [ |:   |
           ]  ]  ] ]\ _ /[ [  [  [ |:   |
           ]  ]  ] ] (#) [ [  [  [ :===='
           ]  ]  ]_].nHn.[_[  [  [
           ]  ]  ]  HHHHH. [  [  [
           ]  ] /   `HH("N  \ [  [
           ]__]/     HHH  "  \[__[
           ]         NNN         [
           ]         N/"         [
           ]         N H         [
          /          N            \
         /           q,            \
        /                           \
```

Regards,
Xavier

**Comment by nickw444** [comment]
Hey there, I just took a look now, and found that the project wasn't even inside my read-the-docs account. Weird! I've re-added it and setup the build plan. 

You should be able to access the docs at the same location now. Let me know if there's anything i've overlooked. Cheers.

**Comment by xavierhardy** [comment]
When I try to fetch the documentation for the latest version (0.9.9), the documentation page title says 0.9.4. I don't know whether it's just the title that is wrong, or whether the documentation is not up-to- date.

**Comment by nickw444** [comment]
That's odd. I've just looked into this, and actually found that the version was also being set to an old value inside `conf.py` for the documentation. The documentation was up to date however.

See the fix here: c0e235f926199f939155eeaa086f1034f1c574ea

All documentation from now onwards should show the [correct version](http://flask-ldap3-login.readthedocs.org/en/latest/)

Thanks helping by reporting this issue :)
